cmd/bb_worker/main.go

```diff
@@ -265,6 +265,11 @@ func main() {
 						log.Fatal("Failed to marshal worker ID: ", err)
 					}
 
+					env := map[string]string{}
```

Comment:
Is this loop really needed? Couldn’t you just pass the original map along?

Reply:
Yeah I wasn't sure, if the main map is copied by value, then skip the loop and initialize env as a copy of runnerConfiguration.EnvironmentVariables?  Not quite a go sage yet.

Reply:
It should be safe to just pass it along. We don't mutate any of the structs we parse from the config.

Reply:
Yes.  I was confusing it for the use inside the Executor where we do need to make a copy to modify it with the action-specified environment